src/module.cpp

```diff
@@ -14,8 +14,12 @@ static void init(v8::Local<v8::Object> target) {
     // expose hello_async method
     Nan::SetMethod(target, "hello_async", standalone_async::hello_async);
 
-    // expose hello_world class
-    HelloWorld::Init(target);
+    // expose HelloObject class
+    object_sync::HelloObject::Init(target);
+
+    // Notice there are multiple "hello" functions as part of this module:
+    // 1) standalone::hello
+    // 2) HelloObject.hello()...not really sure why this function doesnt have to be expliclty exposed
```

Comment:
Yes, that is right: the `HelloObject` object/class gets exposed here with `object_sync::HelloObject::Init` and the `HelloObject.hello()` function gets exposed by the `SetPrototypeMethod(fnTp, "hello", hello);` [here](https://github.com/mapbox/node-cpp-skel/pull/48/files#diff-f8bbad9fb9cba460cdc45ebc483664e0R42)